tier1/checklist.md

```diff
@@ -0,0 +1,387 @@
+|-------------|------------------------------------------------------------------|-----------------------------------------|
+| gitleaks | Open source tool that detects and prevents secrets (passwords/api/ssh keys) checked-in to your git repo | https://github.com/gitleaks/gitleaks   https://akachandwani.medium.com/what-is-gitleaks-and-how-to-use-it-a05f2fb5b034 |
+| git filter-repo    | Entirely remove unwanted files / files with sensitive data from a repository's history                     |    https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository                   |
+
+
+
+---
+
+
+### Review Repository Hygiene
+
+As part of our repository hygiene requirements, the project must include certain files and sections. Using repolinter will help identify missing files and content that will need to be added to your repository before outbounding.
+
+**Running repolinter on your repository**
+
+[^1]: the repolinter.json seems like some sort of link to something
+
+
+
+1. Add [repolinter.json](https://github.com/DSACMS/repo-scaffolder/blob/main/tier1/%7B%7Bcookiecutter.project_slug%7D%7D/repolinter.json) to the root directory of your project
+
+[^1]: that command is highlighted green could also think to something? - [x]
+
+2. Run command 
+```
+repolinter lint .
+```
+3. The result produces a list of files section existence checks, indicating whether each requirement was met or not.
+
+Which will look something like this:
+
+![repolinter results](../assets/repolinter-results.png)
+
+**Running repolinter on your repository via GitHub Actions**
+
+1. Add the tier-specific [checks.yml](https://github.com/DSACMS/repo-scaffolder/blob/main/tier1/%7B%7Bcookiecutter.project_slug%7D%7D/.github/workflows/checks.yml) to the .github directory of your project. The file includes a job that runs a repolinter called [repolinter-checks](https://github.com/DSACMS/repo-scaffolder/blob/main/tier1/%7B%7Bcookiecutter.project_slug%7D%7D/.github/workflows/checks.yml#L13).
+
+2. Manually trigger the workflow
+
+3. The results produces an issue on the repository with a list of file and section existence checks, indicating whether each requirement was met or not.
+
+
+ **Review Content**
+
+ The project should include the following files and sections [(link to template)](https://github.com/DSACMS/repo-scaffolder/tree/main/tier1/%7B%7Bcookiecutter.project_slug%7D%7D):
+
+ - [ ] **README.md**
+    _An essential guide that gives viewers a detailed description of your project_
+
+    ---
+
+
+| **Section**         | **Description**                                                                                                                                                                                                                                                                                | **Included** |
```

Comment:
Please add a ✅  and ❌ emoji like below
<img width="430" alt="Screenshot 2024-09-25 at 1 51 36 PM" src="https://github.com/user-attachments/assets/bca0b070-2e93-4cea-a9d8-aa18928042ad">

Reply:
I have added these emojis but they are side by side instead of on top of each other